bin/wmagent-couchapp-init

```diff
@@ -367,6 +367,10 @@ if __name__ == "__main__":
     
     if hasattr(wmagentConfig, "Tier0Feeder"):
         installCouchApp(wmagentConfig.JobStateMachine.couchurl, wmagentConfig.Tier0Feeder.requestDBName, "T0Request")
+        centralWMStatsURL = config.AgentStatusWatcher.centralWMStatsURL
```

Comment:
Copy&Paste error ? This should be wmagentConfig.AgentStatusWatcher.centralWMStatsURL, right ?

Reply:
Thanks. updated now